src/final/hooks/use-user-widgets.ts

```diff
@@ -31,17 +49,27 @@ export const useUserWidgets = () => {
     enabled: gitcoinEnabled && isTwitter,
   });
 
-  if (isWarpcast) {
-    return {
-      // widgets: userWidgetDataAdapter.fromScrapedUsers({ users }), // TODO: uncomment to enable user widgets on Warpcast
-      widgets: [],
-    };
-  }
+  if (isWarpcast || isSupercast) {
```

Comment:
what do you think about moving this condition to some variable and reusing it in the query and here?